app/javascript/components/maintaining/SubmissionsSummaryTable.tsx

```diff
@@ -23,11 +36,15 @@ function SubmissionsSummaryTableRow({
   )
 }
 
-export function SubmissionsSummaryTable({ submissions }) {
+export function SubmissionsSummaryTable({
+  submissions,
+}: {
+  submissions: Submission[]
+}) {
   const [stateSubmissions, setSubmissions] = useState(submissions)
 
   useEffect(() => {
-    const received = (data) => {
+    const received = (data: SubmissionFromChannel) => {
```

Comment:
I haven't looked into actioncable's typing yet, but assuming this `data` param is _actual data_, and not an `event` param with the data merged in? If yes, this is fine. If no, I recommend using the `XXXEvent` convention for naming.